src/js/view/components/timeslider.js

```diff
@@ -193,11 +205,23 @@ define([
                     timetipText = 'Live';
                 }
             }
-            this.timeTip.update(timetipText);
+            var timeTip = this.timeTip;
+
+            timeTip.update(timetipText);
             this.showThumbnail(time);
 
-            utils.addClass(this.timeTip.el, 'jw-open');
-            this.timeTip.el.style.left = (pct*100) + '%';
+            utils.addClass(timeTip.el, 'jw-open');
+
+            var timeTipWidth = timeTip.width();
```

Comment:
It looks like this isn't working correctly for the individual thumbnails test on http://player-pr-test-jenkins.longtailvideo.com/builds/5277/archive/test/squash/bootstrap.html?primary=html5&edition=ads&feature=cuepoints_thumbnails&timeSliderAbove=true

Does this need to be using the jw-overlay that the tooltip contains to determine its size?  On that page, the tooltip width is much smaller than expected while its child overlay element is at the correct size.